### PR TITLE
SGMC matrix update and pointing mount controller crate to local folder

### PR DESCRIPTION
**Pull request by rodrigo-romano**
The most relevant changes are the SGMC updates in the fem client (`discrete_state_space.rs`).
There is also a change in the mount client `Cargo.toml` to enable tests using a local controller code version.

### CODE REVIEW

File: clients/mount/Cargo.toml
Comment (rconan) [inline clients/mount/Cargo.toml:1]: can you remove the path in the `gmt_mount-ctrl_controller` dependency?

**Review by rconan — CHANGES_REQUESTED** [review]

File: clients/mount/Cargo.toml
Comment (rodrigo-romano) [reply clients/mount/Cargo.toml:1]: Done. Originally, in my fork, the mount controller dependency pointed to version 1.0.0. I used the latest version in crates.io, namely 1.1.0.

Comment (rconan) [reply clients/mount/Cargo.toml:1]: Thanks

**Comment by rconan** [comment]
I have modified the code to handle the case where the ground acceleration input is missing

**Review by rconan — APPROVED** [review]

**Pull request merged**